spring-cloud-starter-dataflow-server/pom.xml

```diff
@@ -46,6 +46,12 @@
 			<artifactId>spring-cloud-dataflow-rest-client</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.security.oauth</groupId>
```

Comment:
Is there a way to avoid using this in tests?

Reply:
And version dep needs to go to main pom.